internal/distributed/proxy/httpserver/request_v2.go

```diff
@@ -141,18 +141,28 @@ type CollectionDataReq struct {
 
 func (req *CollectionDataReq) GetDbName() string { return req.DbName }
 
+type searchParams struct {
+	// not use metricType any more, just for compatibility
+	MetricType    string                 `json:"metricType"`
+	Params        map[string]interface{} `json:"params"`
+	IgnoreGrowing bool                   `json:"ignore_growing"`
```

Comment:
Do we need to maintain consistency in naming conventions around here?
metric_type --> metricType
so ignore_growing --> ignoreGrowing?